typings.d.ts

```diff
@@ -0,0 +1 @@
+declare module '*.png';
```

Comment:
Will need to delete this as we are not utilizing this in tsconfig.json anymore

Reply:
Should be covered here: https://github.com/NEAR-Edu/stats.gallery-server/pull/18/files#diff-b55cdbef4907b7045f32cc5360d48d262cca5f94062e353089f189f4460039e0R11

Reply:
ohhhh, that makes sense. alrighty thank you for pointing that out